src/main/java/net/sf/jabref/gui/actions/Actions.java

```diff
@@ -11,7 +11,6 @@
     public static final String ADD_TO_GROUP = "addToGroup";
     public static final String AUTO_SET_FILE = "autoSetFile";
     public static final String BACK = "back";
-    public static final String CHECK_INTEGRITY = "checkIntegrity";
```

Comment:
The thing is that it only makes sense if the new Action inherits BasicAction, which is a private class in JabRefFrame. Then it just is stored in a field. Since there seems to be little reason storing it in an unused field in the new Action and it was only used there, I didn't see any point in keeping the String.